tools/js-sdk-release-tools/src/test/version/version.test.ts

```diff
@@ -57,3 +61,26 @@ describe("Get latest stable version after GA but beta version before GA", () =>
         },
     );
 });
+
+describe("Used Versions", async () => {
+    test("Get used versions from npm view", async () => {
+        const view = {
+            versions: {
+                "3.0.0-alpha.20250619.1": {
```

Comment:
do we have issues for stable versions? i notice the description mentioned the impacted scopes are only for preview?

Reply:
I haven't encountered this issue in for stable versions yet